src/components/MainContent/TaskModel.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { RenderListItems, Stem, Task } from './index';
+import { Content } from './Content';
+import { MainHeader, NonBulletList, NumberList, Passage, Section, SubHeader } from './Components';
+
+interface TaskModelProps {
+  task: Task;
+  stems: Stem[];
+  evidences: string[];
+  scoringRule: string;
+  index: number;
+}
+
+export const TaskModel = ({ task, stems, evidences, scoringRule, index }: TaskModelProps) => {
+  const taskPrefix = `taskModel${index}`;
+  const desc = Content(task.taskDesc);
+
+  const RenderStemsBy = (target: string, stems: Stem[]) =>
```

Comment:
On top of making-this camel-cased, I would also move this out of the component's scope, so that we're not re-instantiating it with every `TaskModel`.